### DISCUSSION TITLE
How to get a model from view?

### DISCUSSION THREAD

**Discussion by Calvinli0769** [prisma/prisma#16173] [Q&A]
I'm using mysql, and how to pull models from views?

**Comment by nurul3101** [comment]
Hello @Calvinli0769 
Prisma doesn't have native support for Views, we have a feature request for it: #678 
It would help our product team prioritise it if you can add a 👍 to the request.
As a workaround, you can manually add that view to `schema.prisma` as a model and then query as you do for normal models.

**Reply by vladi-stevanovic** [reply]
👋 Hi , did you perhaps have a chance to check Nurul's recommendation? 

As a note, we are currently designing our Views support in Prisma ORM and could use your help - There is an active survey running on this topic where we are seeking for feedback. You can find more details here if you are interested: [#678 (comment)](https://github.com/prisma/prisma/issues/678#issuecomment-1330304851)

Or, if you've found a solution, we'd love to know how you solved this problem. Feel free to post it in this thread and "Mark as answer" if this discussion is now solved!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
